Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Zygote"
 uuid = "e88e6eb3-aa80-5325-afca-941959d7151f"
-version = "0.4.6"
+version = "0.5.0"
```

Comment:
I think we can keep to a patch version; this is really more of a repl feature that packages don't depend on.

Reply:
Don't do it, follow SemVer.
Don't chose to do the bad thing.
We have semver ant it is a nice thing.

Either deprecate or mark breaking

Reply:
The thing about semver is that there's no well-defined boundary for what's breaking in Julia. Essentially any change could be breaking (including adding new exports and renaming or refactoring private functions), so in general we have to make a judgement about what will break in practice.

So I think a positive argument is needed for why this should be treated differently from an internal or interactive-usage-only function, if it should be.